code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm

```diff
@@ -1,6 +1,7 @@
 #define TUMOR_INACTIVE 0
 #define TUMOR_ACTIVE 1
 #define TUMOR_PASSIVE 2
```

Comment:
As a side note. Can you undef these as well?

Reply:
Done